### DISCUSSION TITLE
Separate Evaluation Datasets

### DISCUSSION THREAD

**Discussion by svenschultze** [axolotl-ai-cloud/axolotl#1163] [Q&A] [answered]
I am training models on a mixture of datasets from different tasks, which I combine using an external script that creates conversations based on these building blocks. To evaluate the models during training, I need to define a separate evaluation dataset that contains building blocks that are not in the training dataset.

My Question:
Is it possible to load a separate eval dataset instead of taking a fraction of the training dataset with axolotl?

**Comment by Taytay** [comment, marked as answer]
It looks like it is now thanks to this new commit:
https://github.com/OpenAccess-AI-Collective/axolotl/commit/5787e1a23f61be093fc26067f8a05342fbdfbe1a
